### ISSUE TITLE
Support require fallback for strange environments

### ISSUE THREAD

**Issue by bmeck** [substack/node-browserify#624]
See: https://twitter.com/rwaldron/status/430414189413138433

Basically: there is a need to allow non-browserifieds modules to defer to another `require` function. In this example @rwaldron wanted to use `node-serialport` from `node-webkit`'s require inside of a `browserify` bundle.

Solution: --require-fallback $name

Make a CLI option to fallback to an environment's require function if we did not bundle that file.

**Comment by terinjokes** [comment]
As noted, the standard prelude already calls to a global require if a module can't be found. You should be able to implement a global require before including the browserify bundle that delegates to any (sync) source that you want, and have browserify exclude bundling the modules you don't want to bundle.

Does this resolve the issue for everyone?

**Comment by moll** [comment]
I don't believe there's a way to ignore a whole module with its files currently if you don't have a local copy of it.  Something like `--external some-module/*` tries to glob at compile time and then still fails given a `require("some-module/some-file")`.
